### PR TITLE
update broken stories

### PR DESCRIPTION
**Pull request by manuelpuyol**

### CODE REVIEW

File: stories/primer/dropdown/menu_component_stories.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "primer/dropdown/menu_component"
```
Comment (manuelpuyol) [inline stories/primer/dropdown/menu_component_stories.rb:3]: for some reason, the story can't find `Primer::Dropdown::MenuComponent` without this require

**Review by joelhawksley — COMMENTED** [review]
Is there a failing test we could write that would have caught this problem?

File: stories/primer/label_component_stories.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@ class Primer::LabelComponentStories < ViewComponent::Storybook::Stories
     controls do
       title "this is a label"
       # TODO: Update to use functional scheme mappings
```
Comment (srt32) [inline stories/primer/label_component_stories.rb:9]:
```suggestion
```

remove this now?

**Comment by manuelpuyol** [comment]
> Is there a failing test we could write that would have caught this problem?

we could run `bin/rails view_component_storybook:write_stories_json` on CI and see if it breaks?

**Review by srt32 — APPROVED** [review]

**Pull request merged**